packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -254,4 +254,14 @@ $DropDown-item-height: 36px;
 .divider {
   height: 1px;
   background-color: $dividerColor;
+}
+
+.span{
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  min-width: 100px;
```

Comment:
won't setting `min-width` be a problem if our dropdown width itself is less that 100px?